csvcubed-devtools/csvcubeddevtools/helpers/file.py

```diff
@@ -0,0 +1,17 @@
+from pathlib import Path
+
+
+def get_test_cases_dir(start_dir: Path = Path(".")):
```

Comment:
I've moved this into its own folder so it can more easily (and sensibly) be referenced by other projects.